Flutter/cartesian-charts/methods.md

```diff
@@ -653,6 +813,336 @@ The [`hide`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/char
 {% endhighlight %}
 {% endtabs %}
 
+### Events in crosshairBehavior
+
+Provided the following methods for handling pointer and gesture events and allowing customizations for various pointer events such as long-press, tap, double-tap, pointer enter, and exit.
+
+* `handleEvent` - Specifies to customize the necessary pointer events.
+* `handleLongPressStart` - Specifies to customize the pointer event when a long-press begins.
+* `handleLongPressMoveUpdate` - Specifies to customize the pointer event during movement after a long-press.
+* `handleLongPressEnd` - Specifies to customize the pointer event when the pointer stops contacting the screen after a long-press.
+* `handleTapDown` - Specifies to customize the pointer event when a tap has contacted the screen once.
+* `handleTapUp` - Specifies to customize the pointer event when it has stopped contacting the screen after a tap.
+* `handleDoubleTap` - Specifies to customize the pointer event when a tap has contacted the screen twice.
+* `handlePointerEnter` -Specifies to customize the pointer event when the mouse enter on the screen.
+* `handlePointerExit` - Specifies to customize the pointer event when the mouse exit on the screen.
+
+This following code snippet defines how to enable/disable specific pointer events and customize to show crosshair in double tap and disabled crosshair showing in pointer enter and exit.
```

Comment:
This following code sample defines how to enable or disable specific pointer events and customize to show crosshair in double tap and disabled crosshair showing in pointer enter and exit.

Reply:
Used the following content for crosshair "This following code sample defines how to enable or disable specific pointer events and customize to show crosshair in double tap and disabled crosshair showing in pointer enter and exit."